src/nrncvode/netcvode.cpp

```diff
@@ -1625,42 +1639,57 @@ bool NetCvode::init_global() {
                     // singly linked list built below
                     int j;
                     for (j = 0; j < ml->nodecount; ++j) {
+                        auto offset = ml->get_storage_offset() + j;
+                        // for each Memb_list instance constructed, keep
+                        // track of its initial storage offset (i.e. offset)
+
                         int inode = ml->nodelist[j]->v_node_index;
+                        int icell = cellnum[inode];
```

Comment:
Nit: is this used anywhere inside of this loop (see below comments)? If not, I think it can be removed.

Reply:
I'm not using it, but I think I should in the succeeding 5 places where ```cellnum[inode]``` is used.